CONTRIBUTING.md

```diff
@@ -11,6 +11,53 @@ composer install && ./vendor/bin/phpunit
 -   [Make a pull request](https://help.github.com/articles/using-pull-requests)
 -   Thanks!
 
+Testing
+=======
+
+Unit tests
+----------
+
+To run the unit tests, install the dependencies with Composer:
+
+```sh
+$ composer install
+```
+
+Then run the Composer "test" script:
+
+```sh
+$ composer test
+```
+
+End-to-end tests
+----------------
+
+These tests are implemented with our notifier testing tool [Maze Runner](https://github.com/bugsnag/maze-runner). This requires a recent version of Ruby to run
```

Comment:
I think it's Ruby 2.5+ that's needed currently.

Reply:
I think I'm going to leave this vague so that we don't have to update it if we drop support for older rubies — I don't really expect non-Bugsnag employees to be running these often anyway